sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -580,6 +613,39 @@ gather_rgba(const T *p, simd<Toffset, N> offsets, simd_mask<N> mask = 1) {
       addrs.data(), mask.data());
 }
 
+/// @anchor usm_gather_rgba
```

Comment:
Here my [comment](https://github.com/intel/llvm/pull/7411/files/7e5eb5cc23dfbf677a8c0d07e659943e06b3806b..9a4a2b22572ef41d080b7987ad0effaf683b1864#r1035496646) above is especially relevant.